modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]AllowUsersToDefineAPIApplicationsInTheAPIBuilder/EditAPIApplication.testcase

```diff
@@ -29,6 +29,245 @@ definition {
 		}
 	}
 
+	@priority = 4
+	test CanEditSchemaNameAndDescriptionInPublishedAPIApplication {
+		property portal.acceptance = "true";
+		property test.liferay.virtual.instance = "false";
+		property test.run.type = "single";
+
+		task ("Given create an published API application with related schema") {
+			APIBuilderUI.switchToRelatedEntryInEditApplication(
+				entity = "Schemas",
+				title = "App-test");
+
+			Click(locator1 = "Button#PLUS");
+
+			Type(
+				locator1 = "TextInput#NAME",
+				value1 = "Name default");
+
+			Type(
+				locator1 = "TextInput#DESCRIPTION_FIELD",
+				value1 = "Description default");
+
+			Click(locator1 = "ObjectAdmin#OBJECT_DROPDOWN");
+
+			Click(
+				key_value = "APIApplication",
+				locator1 = "ObjectAdmin#ADD_ENTRY_DROPDOWN");
+
+			Click(locator1 = "APIBuilder#CREATE_BUTTON");
+
+			Alert.viewSuccessMessage();
+
+			Click.pauseClickAt(locator1 = "APIBuilder#PUBLISH_BUTTON");
+		}
+
+		task ("And Given edit the schema in Edit API Application > Schemas") {
+			APIBuilderUI.goToEditAPIApplicationPage(key_title = "App-test");
```

Comment:
There is an existing maco which can switch to any tab in edit api application, please check and update
`APIBuilderUI.switchToRelatedEntryInEditApplication(`

Reply:
Done! Thanks for this review @yanan. Was necessary to create a new macro because there are parts from `switchToRelatedEntryInEditApplication` that I needn’t